.enterprise-commit

```diff
@@ -1 +1 @@
-ee1dea24e8808e08167174b93e5faf34f06cc88d
+580ffe3aa8e27e390a3ab7845880639ce4be62f7
```

Comment:
@lvrach is this OK? I had 3 failing checks so I had to update the enterprise version to make them pass.

Reply:
Yes, this needs to point to the related enterprise commit